core/src/style/pointStyle.h

```diff
@@ -50,6 +53,8 @@ class PointStyle : public Style {
 
     UniformLocation m_uTex{"u_tex"};
     UniformLocation m_uOrtho{"u_ortho"};
+
+    mutable std::unique_ptr<LabelMesh> m_mesh;
```

Comment:
It will be good to have `m_mesh` be renamed to avoid confusion with `meshes` owned by `tiles`.

I do see the point of `TextStyles` and `PointStyles` owning mesh(es), but it will be good to clarify with the "names" on how these style meshes are different than the ones in `Tile::m_geometry`.